src/web/app/pages-static/index-page/index-page.component.scss

```diff
@@ -5,3 +5,8 @@ img {
 .margin-bottom-10px {
   margin-bottom: 10px;
 }
+
+.center {
```

Comment:
The class name is too generic. You could rename it to something like `.button-center` as different html tags require different ways to center, e.g. `.item-center`.